src/components/event/experimental-api/ExperimentalAPIEventStore.ts

```diff
@@ -130,6 +140,16 @@ export class ExperimentalAPIEventStore {
 		this.isExpanded.set(eventId, !this.isExpanded.get(eventId));
 	};
 
+	@action clearEventChildren(eventId: string) {
+		if (eventId === undefined) return;
```

Comment:
I don't think we even need this function. we definetely dont want to reset parentsChildrenMapMap. btw can you rename it and remove extra Map word?